lib/Dialect/TritonGPU/Transforms/CombineTensorSelectAndIf.cpp

```diff
@@ -58,6 +77,8 @@ class CombineTensorSelectAndIfPass
           continue;
         if (canMergeIntoIf(selectOp, ifOp, dom)) {
           selectToIf[ifOp].push_back(selectOp);
+          // Only collect the first user of scf.if, as canonicalize will
+          // combine scf.if ops with the same condition.
```

Comment:
The comment is not clear to me. You didn't collect any users here

Reply:
I have removed these comments